crates/typst-library/src/layout/spacing.rs

```diff
@@ -64,9 +62,19 @@ impl Behave for HElem {
         }
     }
 
-    fn larger(&self, prev: &Content) -> bool {
-        let Some(prev) = prev.to::<Self>() else { return false };
-        self.amount() > prev.amount()
+    fn larger(
+        &self,
+        prev: &(Content, Behaviour, StyleChain),
```

Comment:
This type is probably not ideal, especially because `Behaviour` is not used. I only used it because it made calling the function easier.